xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.464.xml

```diff
@@ -107,7 +108,13 @@
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
+            <lb n="1"/> <persName><gap reason="lost" extent="unknown" unit="character"/>inius <persName><name type="praenomen" key="Lucius"><expan><abbr>L</abbr><ex>uci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>ibertus</ex></expan> <name type="cognomen" key="Gamus">Gamus</name></persName>
+            <lb n="2"/> <supplied reason="lost">hic es</supplied>t ille situs qu<surplus>e</surplus>i
+            <lb n="3"/> <supplied reason="lost">quali</supplied>s amicus amico
+            <lb n="4"/> <supplied reason="lost">quoqu</supplied>e indicio. <expan><abbr>V</abbr><ex>ixit</ex></expan> <expan><abbr>a</abbr><ex>nnos</ex></expan> <num value="44">XXXXIV</num>.         
```

Comment:
Double check the number -- I'm not sure I see the vertical stroke before the V in the image I have.